### DISCUSSION TITLE
Crash during new project loading screen

### DISCUSSION THREAD

**Discussion by wrshield** [stride3d/stride#2253] [Newcomers]
Hello,

I'm new to the Stride engine and hope to contribute to the project by working on one of the smaller issues.

I'm trying to build and run the latest version of Stride, but after cloning the repo and following the instructions in the readme, when loading a new project the loader crashes with no error.  It seems to always crash at the "Restore NuGet packages for MyGame.Windows" step.

![Loader_crash](https://github.com/stride3d/stride/assets/145876802/f17ed941-a8df-440e-af5e-bfefcc02e4fa)

After that, when attempting to load the project file that was created, I got this error:

_Error: There was a problem opening the solution.
ReflectionTypeLoadException: Unable to load one or more of the requested types.
Could not load type 'Stride.Games.GameForm' from assembly 'Stride.Games, Version=4.2.0.1, Culture=neutral, PublicKeyToken=null'._
(there is more output in the error message but this is cropped for brevity)

I was able to get the project to build and run successfully by going back to the release 4.2.0.2122, and I'm wondering if that is the latest stable version?  Or is there some other workaround to use the latest build from GitHub?

Thanks!

**Comment by Jklawreszuk** [comment]
Hi, We always glad to see new hands on the board 🙌 Intereseting... Are you sure you are using the latest master branch ? Btw, I encourage you to join on our the discord 👌

**Comment by Doprez** [comment]
This was the error I was getting before one of the PRs was reverted. https://github.com/stride3d/stride/commit/ad8496ffbdd77d1a09e0fbf75a760c493ef1a599 can you make sure your branch is up to date?

If it continues let us know.

**Reply by wrshield** [reply]
I'm back to the latest branch.  It seems to be intermittent now.  Sometimes it gets into a state where it can't load a project, but doing a Clear All NuGet Storage then re-building seems to get it to work again.  Not sure what causes it to no longer be able to load a project though.  But, right now it's working for me.

**Reply by Doprez** [reply]
I know I have had issues with the PDB files being in use while using the debug build of Stride GameStudio. Its annoying but I dont know of a work around for it.
